emulator/setup.c

```diff
@@ -48,6 +48,10 @@ void setup(void) {
 	setup_flash();
 }
 
+void __attribute__((noreturn)) shutdown(void) {
+	for(;;);
```

Comment:
`for(;;) pause();` seems more appropriate here.

Reply:
Fixed that.  But in the long run, we may think of reworking also emulatorPoll, to not hog the CPU in normal mode.